src/Parse/Lexer.x

```diff
@@ -84,7 +84,9 @@ tokens :-
   \>\=                                  { mapToken (\_ -> TokenRightChevronEq) }
   \<\=                                  { mapToken (\_ -> TokenLeftChevronEq) }
   \!                                    { mapToken (\_ -> TokenExclamationMark) }
-  \"($printable # \")*\"                { mapToken (\s -> TokenStr (sanitizeStr s)) }
+  \"($printable # \")+\"                { mapToken (\s -> TokenStr (sanitizeStr s)) }
```

Comment:
So now we have the three of them ? Do we want to stick with this or do we want to stick to one, potentially two ?

Reply:
Happy to keep that for now as it's very simple to revert anyways.

Reply:
Yeah, for low-friction porpoises I think supporting what JS supports helps us here.